### PR TITLE
Add/admin page

### PR DESCRIPTION
**Pull request by j-chimienti**

### CODE REVIEW

**Comment by giftedprimate** [comment]
Ah snap! Looks like my prune job wrecked this. If you don't mind can you pull this down and see if you can address the conflict? I essentially removed everything not related to authentication and the web sockets are handled by their respective control file in services now.

After star-system is merged in these problems should stop. There was just no way to accomplish this refactor without effecting EVERYTHING! ;)

**Comment by j-chimienti** [comment]
screen shot 2018-11-21 at 9 02 50 am

init signup calendar

File: app/daos/PlayerAccountDAO.scala [hunk 1]
```diff
@@ -31,6 +32,7 @@ class PlayerAccountDAO @Inject()(
   private val timesAccessed = 'timesAccessed
   private val maxLevel = 'maxLevel
   private val maxStep = 'maxStep
+  private val createdLabel = 'created
 
   private val codecRegistry = fromRegistries(
     fromProviders(aProviders: _*),
```
Comment (j-chimienti) [inline app/daos/PlayerAccountDAO.scala]: I need to fix codecRegistry

File: app/daos/PlayerAccountDAO.scala [hunk 2]
```diff
@@ -77,4 +98,69 @@ class PlayerAccountDAO @Inject()(
...
+  //        )
+  //
+  //      )).toFuture()
+  //      //.map(_.toString)
+  //    }
+
+
+  def signupsPerDay: Future[Seq[UserAccountSignups]] = {
+
+    val signupsPerDayQuery =
```
Comment (j-chimienti) [inline app/daos/PlayerAccountDAO.scala]: Having issues w/ scala driver, grouping _id by more than one field


```
[MongoCommandException: Command failed with error 15983: 'An object representing an expression must have exactly one field: { $month: "$created", $dayOfMonth: "$created", $year: "$created" }' on server localhost:27017. The full response is { "ok" : 0.0, "errmsg" : "An object representing an expression must have exactly one field: { $month: \"$created\", $dayOfMonth: \"$created\", $year: \"$created\" }", "code" : 15983, "codeName" : "Location15983" }]
```

Thanks @console-rastling I went w/ raw Query and works

File: app/daos/PlayerAccountDAO.scala (hunk 1 above)
Comment (giftedprimate) [reply app/daos/PlayerAccountDAO.scala]: I think to fix the codec registry you just need to add your `SignupDates` and `UserAccountSignups` case classes to the registry.

Comment (j-chimienti) [reply app/daos/PlayerAccountDAO.scala]: Ok, I added the case classes to the registry and removed duplicate, but there is still duplicate collection reference

**Comment by giftedprimate** [comment]
Hi @j-chimienti, I reverted the code this morning to a working state (008f00ccf7812327760bdb8f8898dcd782dfc0a9) and did a force push. I merged some crappy code in last Friday. Master is working again this morning.

**Comment by j-chimienti** [comment]
Kk cool, master is working for me, and I added button to unlock all levels this morning.

Currently thinking about adding level info to admin page (aggreate over stats collection and sum, avg, max of timesPlayed and wins)

**Comment by j-chimienti** [comment]
closing, and will reopen PR for master branch

**Pull request closed without merging**